core/src/test/java/org/apache/struts2/interceptor/ActionFileUploadInterceptorTest.java

```diff
@@ -205,6 +252,22 @@ public void testAcceptFileWithNoFile() {
         assertTrue(msg.indexOf("inputName") > 0);
     }
 
+    public void testAcceptFileWithNoContent() {
+        interceptor.setAllowedTypes("text/plain");
+
+        ValidationAwareSupport validation = new ValidationAwareSupport();
+        boolean notOk = interceptor.acceptFile(validation, NULL_CONTENT, "filename.html", "text/plain", "inputName");
+
+        assertFalse(notOk);
+        assertFalse(validation.getFieldErrors().isEmpty());
+        assertTrue(validation.hasErrors());
+        List<String> errors = validation.getFieldErrors().get("inputName");
```

Comment:
just a small suggestion to use AssertJ:

        assertThat(validation.getFieldErrors().get("inputName"))
            .hasSize(1).first(STRING)
            .startsWith("Error uploading:")
            .contains("inputName");

Reply:
I want to make this change as small as possible, I will do a large refactoring once cherry-picking changes into Struts 7